typescript/src/schema/proto.ts

```diff
@@ -436,6 +442,35 @@ function augmentService(
       '.google.api.oauthScopes'
     ].split(',');
   }
+  augmentedService.pathTemplate = [];
+  for (const property in messages) {
+    if (messages.hasOwnProperty(property)) {
```

Comment:
It's easier to do `for (const property of Object.keys(messages))` without any additional filtering.

Reply:
got it, will make changes